cat-client/src/main/java/com/dianping/cat/message/spi/internal/DefaultMessageTree.java

```diff
@@ -353,6 +354,10 @@ public String toString() {
 			result = buf.toString(Charset.forName("utf-8"));
 		} catch (Exception ex) {
 			Cat.logError(ex);
+		}finally {
+			if(buf!=null){
+				BufReleaseHelper.release(buf);
+			}
```

Comment:
Please follow our code style while sending PRs.

There should be an empty space on the each side of the preserved keyword in this scenario.